### PR TITLE
Feature/html wireframe

### PR DESCRIPTION
**Pull request by thankgodr**
- Made API Call using async wait
- Submit the form to create a new score to the leaderboard 
- Implement the refreshes button to refresh the list and populate the scoreboard with data from API
- Created a new Game using the API for the new Game.

### CODE REVIEW

File: src/js/controllers/leaderboardnetwork.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(body),
+    });
+    const json = returnJson ? await response.json() : await response.text();
+    return json;
+  };
```
Comment (Dipeshtwis) [inline src/js/controllers/leaderboardnetwork.js:59]: - [ ] I am unable to send the `name` and `score` to the API. I am sending you the screenshot of what it shows.

Screenshot from 2021-10-07 19-24-30

File: src/js/index.js [hunk 2]
```diff
@@ -1,17 +1,57 @@
...
-    const singleScore = document.createElement('li');
-    singleScore.className = loopIndex % 2 === 0 ? 'list-group-item' : 'list-group-item disabled';
-    const textNode = document.createTextNode(`Name: ${element.name} Score: ${element.score}`);
-    singleScore.appendChild(textNode);
-    domScoresHolders.appendChild(singleScore);
+  domScoresHolders.innerHTML = '';
+  leaderboardnetwork.fetchScores().then((outcome) => {
+    const res = JSON.parse(outcome);
+    scoreController.scoresArray = res.result;
+    // scoreController.populateDemo(5);
```
Comment (Dipeshtwis) [inline src/js/index.js:16]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @thankgodr, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used arrow function instead of function keyword.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**